js/summary.go

```diff
@@ -31,20 +32,22 @@ func metricValueGetter(summaryTrendStats []string) func(metrics.Sink, time.Durat
 	return func(sink metrics.Sink, t time.Duration) (result map[string]float64) {
 		switch sink := sink.(type) {
 		case *metrics.CounterSink:
-			result = sink.Format(t)
-			rate := 0.0
-			if t > 0 {
-				rate = sink.Value / (float64(t) / float64(time.Second))
+			result = map[string]float64{
+				"count": sink.LastValue(),
```

Comment:
why not also assign `rate` here, instead of manually assigning `result["rate"]` on the line below?